graphql/bookings.graphql

```diff
@@ -13,10 +13,10 @@ extend type Mutation @middleware(checks: ["auth"]) {
     ): Booking! @field(resolver: "App\\Modules\\Bookings\\CreateBookingHandler")
 
     updateBooking(
-        id: ID!
-        from: DateTime @rules(apply: ["before:to"]),
-        to: DateTime @rules(apply: ["after:from"]),
-    ): Booking @update
+        id: ID! @rules(apply: ["exists:bookings"])
+        from: DateTime @rules(apply: ["required", "before:to"]),
```

Comment:
Should we have `required` or the not null `!` operator on the DateTime type. Not sure which is better tbh

Reply:
Interestingly, the not null operator didn't validate. I could pass in a blank from and it would hit the handler with an undefined index error for the array key.

Reply:
perhaps the `date` validation would be better then, coupled with the not null operator?

Reply:
okay, further investigation:
- not null operator works well
- we aren't using an ISO8601 format because of the scalar defined into `schema.graphql `
- no need for the laravel validation rules beyond the before & after